components/Chips/examples/supplemental/ChipsExamplesSupplemental.h

```diff
@@ -71,12 +66,11 @@
 @end
 
 @interface ChipsInputExampleViewController : UIViewController
-@property(nonatomic, strong) MDCTypographyScheme *typographyScheme;
-@property(nonatomic, strong) MDCSemanticColorScheme *colorScheme;
-@property(nonatomic, strong) MDCShapeScheme *shapeScheme;
+@property(nonatomic, strong) MDCContainerScheme *containerScheme;
```

Comment:
These should be `id<MDCContainerScheming>` and not the concrete type. Although go/mdc-ios-theming is outdated I think the recommendation for accepting the protocol should still be preferred. In our catalog , think we should adopt the recommendation.